tests/console/yast2_cmdline.pm

```diff
@@ -40,8 +42,25 @@ sub run_yast_cli_test {
 }
 
 sub run {
+    my $cmd;
+
     select_console 'root-console';
 
+    if (is_sle) {
+        if (is_sle('15+') and get_var('REPO_SLE15_MODULE_BASESYSTEM_SOURCE')) {
+            $cmd = "ar -f " . "$utils::OPENQA_FTP_URL/" . get_var('REPO_SLE15_MODULE_BASESYSTEM_SOURCE') . " repo-source";
+        }
+        elsif (is_sle('<15') and get_var('REPO_SLES_SOURCE')) {
+            $cmd = "ar -f " . "$utils::OPENQA_FTP_URL/" . get_var('REPO_SLES_SOURCE') . " repo-source";
+        }
+    }
+    else {
+        die 'Repo syncing issue on openSUSE';
```

Comment:
You should to at least reference the progress ticket but I am not happy with introducing a `die` message here making the test fail with an obvious openQA ticket reference. But let's go with it and hope that we will handle the openSUSE part not too late.